### DISCUSSION TITLE
Docker issue: ROOT level privileges prohibited!

### DISCUSSION THREAD

**Discussion by TH-Braemer** [abraunegg/onedrive#1275] [Q&A] [answered]
Hi @abraunegg 
I'm trying to get the OneDrive Docker image to work, but I'm having some startup issues, and I don't have a clue what it's trying to tell me... Could you maybe give me a hint?

I've tried to follow the Docker guide. I would like to use a docker-compose file, but I'm struggling with the initial registration step.
This is what I've done, and the results:
```
sestatus
  SELinux status: disabled
```
```
onedriveConf="/mnt/USBDrive/DataDisk/Docker/OneDrive/Conf"
onedriveDir="/mnt/USBDrive/DataDisk/Docker/OneDrive/Backup"
docker run -it --name onedrive -v "${onedriveConf}:/onedrive/conf" -v "${onedriveDir}:/onedrive/data" -e "ONEDRIVE_UID:1000" -e "ONEDRIVE_GID:1000" driveone/onedrive:latest-v2.4.9

++ stat /onedrive/data -c %u
+ : 0
++ stat /onedrive/data -c %g
+ : 0
++ getent group 0
+ odgroup=root:x:0:
+ odgroup=root
++ getent passwd 0
+ oduser=root:x:0:0:root:/root:/bin/bash
+ oduser=root
+ usermod -g root root
usermod: no changes
+ grep -qv root /dev/fd/63
++ groups root
+ echo 'ROOT level privileges prohibited!'
ROOT level privileges prohibited!
+ exit 1
```
PS: The bullits should be a single '+', and the X's should be an 'x' surrounded by ':', but I couldn't figure out how to stop it from formatting my post

Hope you can help.
Best regards
Thomas

**Comment by abraunegg** [comment, marked as answer]
@TH-Braemer 
> `ROOT level privileges prohibited!`

It appears you are trying to run your Docker containers as 'root'

Please review Docker documentation on how to correctly run Docker containers as a normal user. This is not an issue with this client or Docker image.

See [this google query](https://www.google.com/search?q=should+i+run+a+docker+container+as+root&rlz=1C1SQJL_enAU887AU887&oq=should+i+run+a+docker+container+as+root&aqs=chrome..69i57j0i22i30l3.353j0j7&sourceid=chrome&ie=UTF-8) for further details.

**Reply by BelphegorPrime** [reply]
I run into the same problem, but even starting the container as a normal user does trigger this error. Does it depend on the `ONEDRIVE_UID` and `ONEDRIVE_GID`?

**Reply by abraunegg** [reply]
@BelphegorPrime 
Please review Docker documentation on how to correctly run Docker containers as a normal user. This is not an issue with this client or Docker image.

See [this google query](https://www.google.com/search?q=should+i+run+a+docker+container+as+root&rlz=1C1SQJL_enAU887AU887&oq=should+i+run+a+docker+container+as+root&aqs=chrome..69i57j0i22i30l3.353j0j7&sourceid=chrome&ie=UTF-8) for further details or seek support from Docker.
